handbook/ce/engaging-other-teams.md

```diff
@@ -1,5 +1,170 @@
 # Engaging Other Teams
 
-This is forthcoming (being worked on by [Virginia](https://about.sourcegraph.com/company/team#virginia-ulrich-she-her)) and is meant to consolidate/revise [CE-to-Engineering issue handover process](ce_to_eng_handover.md) and [How to route questions from customers](routing_questions.md). It will also touch on the fact that we never want CSEs to be gatekeepers and have a template for details CSEs will include when seeking help from engineering. And it will include details about engaging marketing, etc.
+Our goal as Customer Support is to only need help from engineering no more than 10% of the time (not counting when we file defects). It's okay that we need help, and we will need help, and we also continually take what we learn each time we do back to the rest of the team (and our product documentation, as relevant).
```

Comment:
What's the rationale behind 10%?  Is just a nice figure to start and iterate with as the team grows?

Reply:
Exactly. Just a starting place based on what I have seen and we will adjust once we have real data to look at!